apix-integrationtests/src/main/java/eu/xenit/apix/tests/search/SearchServiceTest.java

```diff
@@ -213,7 +207,10 @@ private void create1001TestDocs() throws InterruptedException {
                     FileInfo mainTestFolder = createMainTestFolder(companyHomeRef);
                     FileInfo testFolder = createTestFolder(mainTestFolder.getNodeRef(), "testFolderSetOf1001");
                     Map<QName, Serializable> props = new HashMap<QName, Serializable>() {{
-                        put(ContentModel.PROP_DESCRIPTION, DESCRIPTION_SET_OF_1001);
+//                        put(ContentModel.PROP_DESCRIPTION, DESCRIPTION_SET_OF_1001);
+                        put(QName.createQName("http://test.apix.xenit.eu/model/content",
+                                "searchServiceLimitTestProperty"),
```

Comment:
It's worth adding constants for usages of property, namespace, and qualified property string here and below. Either in `apix-integrationtests/src/main/java/eu/xenit/apix/tests/BaseTest.java` or this class (your choice).